tiledb/CMakeLists.txt

```diff
@@ -527,6 +534,14 @@ if ((TILEDB_S3 OR TILEDB_AZURE OR TILEDB_GCS OR TILEDB_SERIALIZATION) AND NOT WI
       endif()
   endif()
 
+elseif( WIN32 AND TILEDB_AZURE )
+  find_package(Curl_EP REQUIRED)
+  include_directories( ${CURL_INCLUDE_DIR} )
```

Comment:
If possible, please use `target_link_libraries` `CURL::libcurl` so that CMake will handle both the includes and linkage correctly. See https://github.com/TileDB-Inc/TileDB/blob/7e1c6df4ee932a2b2393fee07cb821bbb2ca2569/tiledb/CMakeLists.txt#L355-L358

(I believe that target should exist on Windows, but I haven't checked)

Reply:
In this effort's current state of evolution, build/test completes without that path.
Removed.